solar_system_wave3.rb

```diff
@@ -0,0 +1,160 @@
+  end
+
+end
+
+# Instances of each planet
+mercury = Planet.new("Mercury", "3.3011 × 10^23 kg", "gray with white speckles", "2,439.7 km", "heavily cratered", "none", "46 to 70 million km")
+
+venus = Planet.new("Venus", "4.8675 × 10^24 kg", "mostly orange", "6,052 km", "volcanic", "none", "average 108.2 million km")
+
+earth = Planet.new("Earth", "5.97237 × 10^24 kg", "blue green", "6378.1 km", "mostly water", "1", "average 149.6 million km")
+
+mars = Planet.new("Mars", "6.4171 × 10^23 kg", "reddish", "3396.2 ± 0.1 km", "dry and dusty", "2", "average 230 million km")
+
+jupiter = Planet.new("Jupiter", "1.8982 × 10^27 kg", "marbled", "71,492 km", "not solid, mostly hydrogen and helium gas", "69", "average 75 million km")
+
+uranus = Planet.new("Uranus", "8.6810±0.0013 × 10^25 kg", "icy blue", "25,559±4 km", "featureless", "27", "about 3 billion km")
+
+planet_list = [mercury, venus, earth, mars, jupiter, uranus]
+
+my_solar_system = SolarSystem.new(planet_list)
+
+# Method on its own, outside of a class, to check if what the user enters is one of the choices available
+def check_if_user_input_is_a_planet_choice(user_input, planet_list)
+  if user_input == "quit"
+    puts "Goodbye! Thanks for playing!"
+    return
+  end
+  info_about_a_planet = ""
+  planet_list.each_with_index do | planet, i |
+    if user_input == planet.name.downcase || user_input.to_i == i+1
+      # when the user_input matches planet
+      info_about_a_planet = planet.attributes
+      puts info_about_a_planet
+    end
+  end
+  if info_about_a_planet == "" # <- this means we didn't assign any value to info_about_a_planet
+    puts "Oops we couldn't find that planet. Try again."
+  else
+    puts "\nIf you want to learn about another planet, let me know which.\nIf not, type \'quit\' to stop the program."
+  end
+
+end
+
+# This is the first message the user will see
+puts "Welcome to the a simple virtual model of the Solar System.\n Please enter \'r\' if you want to *read* about a planet\n or \'m\' if you want to *make* a new planet."
+first_decision = gets.chomp.downcase
+
+if first_decision == "r"
+  # if user wants to *read* about an existing planet by typing in "r", then check if the user input matches one of the available planets
+  puts "\nYou are now inside the planetary selector...\nPlease choose by entering the name of a planet or its corresponding number:"
+  # this will show the user the list of available planets
+  my_solar_system.summary
+  user_input = gets.chomp.downcase
+
+  check_if_user_input_is_a_planet_choice(user_input, planet_list)
+
+  # ask the user what planet they want to learn about
+  until user_input == "quit"
+    user_input = gets.chomp.downcase
+    check_if_user_input_is_a_planet_choice(user_input, planet_list)
+  end
+
+# if the user wants to *make* their own planet by typing in "m", then prompt them to enter info about that new_planet
+else first_decision == "m"
```

Comment:
You don't need a condition on an `else` statement, only on an `elsif`. In fact, Ruby will ignore this condition entirely, and always execute the `else` clause if the `if` didn't happen.